pdfminer/high_level.py

```diff
@@ -88,3 +90,46 @@ def extract_text_to_fp(inf, outfp,
         interpreter.process_page(page)    
 
     device.close()
+
+
+def pdf_to_text(pdf_file, password="", page_numbers=set(), maxpages=0,
+                caching=True, codec="utf-8"):
+    """
+    Parses and returns the text contained in a PDF file.
+    Takes loads of optional arguments but the defaults are somewhat sane.
+    Returns a string containing all of the text extracted.
+
+    pdf_file: the path to the PDF file to be worked on
+    password: For encrypted PDFs, the password to decrypt.
+    page_numbers: zero-indexed page numbers to operate on.
+    maxpages: How many pages to stop parsing after
+    disable_caching: Does what it says on the tin
+    codec: Text decoding codec
+    """
+
+    rsrcmgr = PDFResourceManager()
+    retstr = StringIO()
+    laparams = LAParams()
+    device = TextConverter(
+        rsrcmgr, retstr, codec=codec, laparams=laparams
+    )
+    fp = open(pdf_file, "rb")
```

Comment:
You could use `with open(pdf_file, 'rb') as fp` here. 

Or is that only Python 3?

Reply:
It also works on python 2.7, fixed that.